### ISSUE TITLE
Openlayers - export raster with banded symbology - fix progressive filename

### ISSUE THREAD

**Issue by andreaordonselli** [tomchadwin/qgis2web#1024]
I have a .tif file that I would like to export with Openlayers theming it by bands like this:

![image](https://user-images.githubusercontent.com/89784373/221842853-50e893e3-1162-4c0a-9552-3e2674440a3f.png)

When I export everything works, except the filename which captures trailing `_4.png`
![image](https://user-images.githubusercontent.com/89784373/221843705-90e25a48-d4f7-4cbb-abfd-bc3b7b28db40.png)

while in layer.js it is called without _4
![image](https://user-images.githubusercontent.com/89784373/221843946-856a232b-ffa0-424f-ba58-e091d075f85c.png)

How can I modify the plugin to **remove _4 from the .png** located in the layers folder?

_PS: If I manually edit the file by deleting _4 from the .png is visibile in map._
![image](https://user-images.githubusercontent.com/89784373/221849949-87471557-67e2-464b-be63-dfbac0e0726d.png)
@boesiii do you have a solution for me?

**Comment by tomchadwin** [comment]
TL;DR Fix the missing suffix, don't remove it.

I don't know why the `_4` isn't being appended in the code - it should be, so it sounds like a bug. The reason for the incremented suffixes is because QGIS projects can have identical layer names, whereas the JS layer objects exported by qgis2web must be unique.

**Comment by andreaordonselli** [comment]
Thanks for the reply.
